mlir/utils/performance/ck-benchmark-driver/ck-benchmark-driver.cpp

```diff
@@ -0,0 +1,224 @@
+//===- ck-benchmark-driver.cpp - Composable Kernel benchmark driver
+//--------===//
```

Comment:
I'd shorten to `CK benchmark driver` and add a longer comment down below so that you don't get header comments spread onto two lines. Or just remove some `-` from the second part.